.github/workflows/conventional_pr_title.yml

```diff
@@ -0,0 +1,23 @@
+name: Conventional PR Title
```

Comment:
If this is a standard template that is going to be copied across various repos does it make sense  to include a comment linking some docs on it?

Reply:
started but miles from fit for purpose 

https://renalregistry.atlassian.net/wiki/spaces/~876730531/pages/2459729921/Systems+team+work+flow

Reply:
Was thinking more in terms of the conventional PR I'm gonna have to look up the docs a bunch of times before I remember what the specific syntax is.

Reply:
or get this

https://marketplace.visualstudio.com/items?itemName=vivaxy.vscode-conventional-commits